atomic_accessor/atomic_accessor.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Atomic Refs Bounded to Memory Orderings & Atomic Accessors"
-document: P2689R1
+document: P2689R2
```

Comment:
Does the title need to be updated as well "Bound[ed]"

Reply:
(They are more occurrences that would need fixing)

Reply:
Fixed (in a few places in the prose as well).